.gitignore

```diff
@@ -24,3 +24,7 @@
 /testnet
 !/testnet/maker_seed
 !/testnet/taker_seed
+
+
+# Clion IDE
+.idea
```

Comment:
Happy to leave this here. You can also exclude local files without changing `.gitignore` if you add them into  `.git/info/exclude`